### DISCUSSION TITLE
Release non-rc version

### DISCUSSION THREAD

**Discussion by Falco20019** [scottoffen/grapevine#153] [General]
`5.0.0-rc.10` is out for 2.5 years now. Could you please publish a non-pre-release? Our CI pipeline enforces [NU5104](https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5104) as error, which means that we can not use pre-releases outside of testing (as suggested by SemVer v2).

**Comment by scottoffen** [comment]
Sure. Let me set up that pipeline.

**Reply by Falco20019** [reply]
Any update on this?

**Reply by scottoffen** [reply]
I'm resolving some dependency issues that came up. It will be a few more days.

**Reply by scottoffen** [reply]
A release version has been published.

**Reply by Falco20019** [reply]
Thank you very much. Really appreciated! 😊
